### PR TITLE
add option to use a different istanbul compatible instrumenter, such …

### PR DESCRIPTION
**Pull request by Swatinem**
…as babel-istanbul

### CODE REVIEW

**Comment by ferlores** [comment]
Thanks for the contribution. Can you please fix the unit tests and maybe create some for the babel-istanbul option?

File: index.js [hunk 1]
```diff
@@ -85,6 +87,11 @@ function writeReports(options) {
 }
 
 module.exports = function (b, opts) {
+  opts = _.extend({
+    instrumenter: 'istanbul',
+    exclude: ['**/node_modules/**', '**/test/**', '**/tests/**'],
```
Comment (ferlores) [inline index.js]: I'm ok with defining instrumented and dir by default, but I prefer not to give defaults for the exclude option

**Comment by Swatinem** [comment]
Done. I think the default exclude option is a matter of opinion, I removed it for now.

**Comment by ferlores** [comment]
This is great, thanks!

**Pull request merged**